spec/object_spec.js

```diff
@@ -111,6 +115,25 @@ describe('Transis.Object', function() {
         });
       });
 
+      describe('with a function as the default option', function() {
+        it('returns the value of calling the function', function() {
+          expect(t.funcDef).toEqual([]);
+        });
+
+        it('invokes the function once per instance', function() {
+          var t2 = new Test;
+          expect(t.funcDef).toBe(t.funcDef);
+          expect(t2.funcDef).toBe(t2.funcDef);
+          expect(t2.funcDef).not.toBe(t.funcDef);
+        });
+
+        it('returns the set value', function() {
+          expect(t.funcDef).toEqual([]);
+          t.funcDef = ['foo'];
+          expect(t.funcDef).toEqual(['foo']);
+        });
```

Comment:
Yeah, thats a good point. With the current implementation you can't actually clear the value as it would always get set back to the original default value. I'll have to take another look at this - thanks.